libcontainer/cgroups/fs/cpuset.go

```diff
@@ -14,6 +15,42 @@ import (
 	"github.com/opencontainers/runc/libcontainer/configs"
 )
 
+var (
+	cpusetLock     sync.Mutex
+	cpusetNoPrefix bool
+	cpusetFastPath bool
+)
+
+func getCpusetFile(path string, name string) string {
```

Comment:
nit: maybe drop the `get` part, making it `cpusetFile` for brevity.

Reply:
done